R/tags.R

```diff
@@ -170,7 +170,12 @@ nullOrEmpty <- function(x) {
 
 # Given a vector or list, drop all the NULL or length-0 items in it
 dropNullsOrEmpty <- function(x) {
-  x[!vapply(x, nullOrEmpty, FUN.VALUE=logical(1))]
+  ns <- lengths(x) == 0
```

Comment:
From a quick look, it seems `lengths()` was added [in R 3.2.0](https://www.rdocumentation.org/packages/base/versions/3.2.0), so we'll want to put `Depends: R (>= 3.2.0)` in the DESCRIPTION

Also, this change would mean `nullOrEmpty` no longer needs to exist (so please remove it)